### DISCUSSION TITLE
806

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#981] [Comments]
# 806

주선님, 유진님과 함께 하는 트레바리 - 독서모임 에서 리더십에 관한 이야기를 하던 중, "리더에게 가장 중요한/필요한 자질은 무엇일까요?" 라는 질문이 나왔다.참여하신 많은 분들이 자신만의 답을 제시하였는데, 나는 "외로움을 버티는 힘 - 팀원에게 미움 받을 용기" 라고 했다.당연히 중요한 리더십 원칙들이 많겠지만, 여러 리더들과 동료들을 보면서 이게 사실은 가장 중요하다는 생각을 했다.계속 팀원의 역할만 하다가 리더가 되신 분들은 팀원들에게서 미움 받는 것에 대해 굉장히 취약한 경우를 종종 봤다.소수의 팀원만 있다 하더라도,해당 팀원들의 리더가 된다는 것은 이전과는 다르게 그들과 어느정도의 거리감을 둬야한다.감정에 치우치지 않는 결정들을 내리기도 해야하고,때론 팀원들이 싫어할만한 이야기도 꺼내야 한다..

https://jojoldu.tistory.com/806

**Comment by jobum97** [comment]
잘 읽었습니다
